### PR TITLE
basic html and css for trait page

### PR DESCRIPTION
**Pull request by KatiaAsh**
relates #7 #23 #28

### CODE REVIEW

**Comment by KatiaAsh** [comment]
Result
result

File: traits.html [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
   <title>Traits</title>
   <link rel="stylesheet" href="traits.css">
 </head>
 
-<body>
+<center>
+  <body>
+    <section class="traitsContainer" id="traitsContainer">
+      <div class="" id="">
+        <h2>Here are the results</h2>
```
Comment (Wyrd00) [inline traits.html]: We already called the immediate feedback 'results'. Maybe change this to "Report of all traits"?

File: traits.html [hunk 2]
```diff
@@ -1,29 +1,35 @@
...
+      </div>
+    </section>
+</center>
+<br>
+<center>
   <div class="traitsSidebar">
+    <img class="image" src="./images/info.png">
```
Comment (Wyrd00) [inline traits.html]: it's repetitive to make a class called 'image' if applied to all img. Consider removing or changing?

File: traits.html [hunk 3]
```diff
@@ -1,29 +1,35 @@
...
     <option>Learning</option>
+    <img class="image" src="./images/chart.png">
```
Comment (Wyrd00) [inline traits.html]: here as well ... and etc etc etc

**Review by Wyrd00 — CHANGES_REQUESTED** [review]

**Review by Wyrd00 — APPROVED** [review]

**Review by tyeung2017 — APPROVED** [review]

File: traits.html (hunk 1 above)
Comment (Karyum) [reply traits.html]: has this been addressed?

**Comment by KatiaAsh** [comment]
@Karyum Hey, can you check this PR and Merge it?
Thanks ~ :heart:

File: traits.html (hunk 1 above)
Comment (Karyum) [reply traits.html]: ??

**Comment by KatiaAsh** [comment]
@tyeung2017 @Wyrd00 @Karyum  I fixed the last issue in trait page
please have a look

**Pull request merged**